zegami_sdk/collection.py

```diff
@@ -344,6 +337,20 @@ def download_single(index, url):
         for i in range(len(images)):
             ordered.append(images[i])
         return ordered
+    
+    def delete_images_with_tag(self, tag='delete'):
+        """Delete all the images in the collection with the tag 'delete'.s"""
+        row_indicies = set()
+        if tag in self.tags.keys():
+            row_indicies.update(self.tags[tag])
+            lookup = self._get_image_meta_lookup()
+            imageset_indices = [lookup[int(i)] for i in row_indicies]
+            c = self.client
+            urls = ['{}/{}/project/{}/imagesets/{}/images/{}'.format(
+                c.HOME, c.API_0, self.workspace_id, self._get_imageset_id(),
+                i) for i in imageset_indices]
+            for url in urls:
+                c._auth_delete(url)
```

Comment:
Would be good to add some output at the end here. Perhaps e.g. `Deleted 9 images`